sktm/patchwork.py

```diff
@@ -25,6 +25,7 @@
 import dateutil.parser
 import requests
 
+from sktm.misc import TestResult
 import sktm.misc
```

Comment:
Do you mean remove the `import sktm.misc`? I think it is not because `join_with_slash` is using it.

Reply:
This is the first commit and `join_with_slash` is not moved to the misc file yet. And if it were, the line should use the same `from` wording

Reply:
I am little bit confused. The `join_with_slash` is in `__init__` file. How I can use the same `from` wording? And one more question is that because it is inside `__init__`, how it can be imported under `sktm.misc`, and then it can use. Could you explain for me?

Reply:
Okay, let's go from the start. The `import sktm.misc` line was there because of the `TestResult`. You changed the uses to say `TestResult` instead of `sktm.misc.TestResult` and added the green-marked line above. But the original `import sktm.misc` line is still present when there's nothing besides `TestResult` to be imported. Does it make more sense now?

Reply:
It makes sense. I did it in the first time whey I try to replace `import sktm.misc` by `from sktm.misc import TestResult`, it has error in `pytest` about `global sktm`. It works fine if I still put `import sktm.misc`. That is reason why I still keep this.

Reply:
For example: 
```
self = <sktm.patchwork.ObjectSummary object at 0x7fc6f5c0cc10>

    def get_mbox_url(self):
        """
            Get the URL pointing at the object's mbox.
    
            Returns:
                URL pointing at the object's mbox.
            """
>       return sktm.join_with_slash(self.url, self.mbox_sfx)
E       NameError: global name 'sktm' is not defined

```

Reply:
Oh. One of the magical things which happen when you have functionality dependent on module importing. If we import the whole file like `import sktm.misc`, then the init file is also loaded. If we import only chosen classes or methods, it is not. Since the `join_with_slash` is in the init file, it got loaded implicitly.

The most robust solution would be to add `import sktm` line instead, which would be then removed as the `join_with_slash` is moved to misc file.

At least now you know why I want to move the functionality out of the init file :) Also, if you run into a similar problem, please say it, instead of hiding it - it can result in a serious problem if we fail to notice it.

Reply:
Thanks. I just do not want to bother you with a small issue.  But I will say the issue instead of hiding it.

Reply:
Sure. A good practice is to try to fix the problem yourself, google to find the reason and how to fix it; and if you still can't figure out what went wrong, then ask. That way you are not "bothering people with small issues" and learning at the same time (and learning how to debug is super important!).